pyamo/service.py

```diff
@@ -3,13 +3,14 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Portions Copyright (C) Philipp Kewisch, 2015-2016
 
-from __future__ import print_function
```

Comment:
I'd like to keep this if possible.

Reply:
I have missed this fixing after 2to3, thank you.